Source/RenderStreamEditor/Private/RenderStreamEditorModule.cpp

```diff
@@ -503,7 +503,12 @@ URenderStreamChannelCacheAsset* UpdateLevelChannelCache(ULevel* Level)
     {
         Cache->SubLevels.Empty();
         for (ULevelStreaming* SubLevel : Level->GetWorld()->GetStreamingLevels())
-            Cache->SubLevels.Add(SubLevel->GetWorldAsset()->GetPackage()->GetPathName());
+        {
+            if (auto WorldAsset = SubLevel->GetWorldAsset())
+            {
+                Cache->SubLevels.Add(WorldAsset->GetPackage()->GetPathName());
+            }
+        }
```

Comment:
I noticed an issue where some sublevels did not have an asasociated world asset
(seemed to be some auto generated world streaming thing)
Since they don't have a world they are not saved as an actual level so we don't care about it.